### PR TITLE
error creation and resolve

### PR DESCRIPTION
**Pull request by labhvam5**

### CODE REVIEW

File: apps/xero/tasks.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 import traceback
 from datetime import datetime, timedelta
 from time import sleep
+from turtle import heading
```
Comment (ashwin1111) [inline apps/xero/tasks.py:6]: what's this?

File: apps/xero/tasks.py [hunk 2]
```diff
@@ -692,6 +714,18 @@ def __validate_expense_group(expense_group: ExpenseGroup):
                 'message': 'Employee mapping not found'
             })
 
+            if employee_attribute:
+                Error.objects.update_or_create(
```
Comment (ashwin1111) [inline apps/xero/tasks.py:718]: why should we update / create? create here is not needed at any case no?

File: apps/xero/tasks.py [hunk 3]
```diff
@@ -730,6 +771,18 @@ def __validate_expense_group(expense_group: ExpenseGroup):
                 'message': 'Category Mapping not found'
             })
 
+            if category_attribute:
+                Error.objects.update_or_create(
```
Comment (ashwin1111) [inline apps/xero/tasks.py:775]: same comment

File: apps/xero/tasks.py (hunk 1 above)
Comment (labhvam5) [reply apps/xero/tasks.py:6]: This is removed in a different PR while adding view .

File: apps/xero/tasks.py (hunk 2 above)
Comment (labhvam5) [reply apps/xero/tasks.py:718]: While validating an expense-group if the employee-mapping is not present then 
- For the first run we have to create the Error in the Error table . 
- For the concurrent runs if the user hasn't resolved the error, then we don't need to create it again if the error is already present. 
Hence we are using update_or_create()

**Review by Shwetabhk — APPROVED** [review]

**Review by ashwin1111 — APPROVED** [review]

**Pull request merged**